landing-pages/site/content/en/blog/airflow-survey/index.md

```diff
@@ -0,0 +1,334 @@
+
+|                                                                     |No.|  %   |
+|---------------------------------------------------------------------|---|------|
+|No - we do not plan to use Kubernetes near term                      | 88|28.57%|
+|Yes - setup on our own via Helm Chart or similar                     | 65|21.10%|
+|Not yet - but we use Kubernetes in our organization and we could move| 61|19.81%|
+|Yes - via managed service in the cloud (Composer / Astronomer etc.)  | 45|14.61%|
+|Not yet - but we plan to deploy Kubernetes in our organization soon  | 42|13.64%|
+|Other                                                                |  7|2.27% |
+
+The most interesting thing is that there’s nearly 30% of users who do not use Kubernetes,
+and they are not going to move. This means we should keep other deployment options in
+mind when working on Airflow 2.0. On the other hand, almost 70% of the users already
+use Kubernetes, or it’s a viable option for them.
+
+**Do you combine multiple DAGs?**
+
+|                                 |No.|  %   |
+|---------------------------------|---|------|
+|No, I don't combine multiple DAGs|127|41.23%|
+|Yes, through SubDAG              | 73|23.70%|
+|Yes, by triggering another DAG   | 72|23.38%|
+|Other                            | 36|11.69%|
+
+In the other category, 9 people explicitly mentioned using `ExternalTaskSensor`,
+and I think it could be treated as running subDAGs by triggering other DAGs.
+
+**Do you use Airflow Plugins? If yes, what do you use it for?**
+
+|                                      |No.|  %   |
+|--------------------------------------|---|------|
+|Adding new operators/sensors and hooks|187|60.71%|
+|I don't use Airflow plugins           |109|35.39%|
+|Adding AppBuilder views & menu items  | 31|10.06%|
+|Adding new executor                   | 18|5.84% |
+|Adding OperatorExtraLinks             |  7|2.27% |
+
+The high percentage - 60%  for “Adding new operators/sensors and hooks” is quite a
+surprising result for some of us - especially that you do not actually need to use the
+plugin mechanism to add any of those. Those are standard python objects, and you can
+simply drop your hooks/operators/sensors code to PYTHONPATH and they will work.
+It seems that this may be a result of a lack of best practices guide.
+
+Plugins are more useful for adding views and menu items - yet only 10%.
+OperatorExtraLinks are even more useful (though relatively new) feature, so it’s not
+entirely surprising they are hardly used.
+
+It was also kind of surprising that someone at all uses plugins to use their own
+executors. We considered removing that option recently - but now we have to rethink
+our approach.
+
+**What metrics do you use to monitor Airflow?**
+
+There were a lot of different responses. Some use Prometheus and other services,
+others do not use any monitoring. One of the interesting responses linked to this
+solution: https://github.com/mastak/airflow_operators_metrics
```

Comment:
Can you remove pure link and replace it with anchor e.g. `[airflow_operators_metrics](https://github.com/mastak/airflow_operators_metrics)`?  Each link should describe the content.
https://uxmovement.com/content/why-your-links-should-never-say-click-here/ 
https://www.w3.org/QA/Tips/noClickHere